lib/ui/views/dish_details/widgets/ingredient_container.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:receipe_app/ui/common/app_colors.dart';
+import 'package:receipe_app/ui/common/ui_helpers.dart';
+import 'package:receipe_app/ui/extension/app_typography.dart';
+import 'package:receipe_app/ui/extension/palette.dart';
+
+/// The custom container that displays ingredients
+class IngredientContainer extends StatelessWidget {
+  const IngredientContainer({super.key, required this.content});
+  final String content;
+  @override
+  Widget build(BuildContext context) {
+    late ThemeData theme = Theme.of(context);
+    AppTypography? typography = theme.extension<AppTypography>();
+    Palette? palette = theme.extension<Palette>();
+
+    return Container(
+      height: screenHeight(context) * .09,
```

Comment:
what's this screenHeight() that passes context ?

Reply:
I saw it on the app, from the ui_helpers
For Mediaquery

Reply:
lovely work @fiki2002 wowed. can you add an hero widget ad productMidel.id tag to the images from parent and child to spice up a bit.